### ISSUE TITLE
Add WooCommerce Mini-Cart

### ISSUE THREAD

**Issue by Misplon** [siteorigin/vantage#228]
Add a WooCommerce Mini-Cart. @adiraomj @AlexGStapleton Where do you guys want to add this? To the header area or the main menu or both as an option?

**Comment by AlexGStapleton** [comment]
By header area, do you mean widget?

**Comment by Misplon** [comment]
Yes, or next to the header text. Being that it has a full size header the menu isn't the only logical place for the mini-cart.

**Comment by AlexGStapleton** [comment]
Hm. It might be easier for us to create a widget and then recommend something like [Widgets in Menu for WordPress](https://wordpress.org/plugins/widgets-in-menu/) for adding it to the menu.

**Comment by Misplon** [comment]
I want to have a mini-cart option in Vantage that is a single checkbox. Done. Added.
